### PR TITLE
Update create-or-update-contact.mjs

### PR DESCRIPTION
**Pull request by daltonlet**

### CODE REVIEW

**Comment by dylburger** [comment]
Thank you so much for submitting this! We've added it to our backlog to review, and our team has been notified.

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/activecampaign/actions/create-or-update-contact/create-or-update-contact.mjs [hunk 1]
```diff
@@ -46,6 +46,11 @@ export default {
       description: "(Deprecated) Please use the DELETE endpoint.",
       optional: true,
     },
+    city: {
```
Comment (lcaresia) [inline components/activecampaign/actions/create-or-update-contact/create-or-update-contact.mjs:49]: Is this case the prop `city` is created, but isn't sent in the request.

Can you check this?

**Review by lcaresia — CHANGES_REQUESTED** [review]
Just a minor fix.

**Comment by vunguyenhung** [comment]
Hello @daltonlet, I will close this PR since there's no update on this for more than 10 days. Kindly create a new Github issue or PR if needed. Thanks!

**Comment by daltonlet** [comment]
Hi! I ended up getting this resolved by contacting your dev team directly. There was an issue with the [{field, value}] pairs on the backend where Pipedream wasn’t recognizing ActiveCampaign's custom fields at all. That was since resolved yesterday! Kindly disregard these changes!

**Pull request closed without merging**